discord-bot/tsconfig.json

```diff
@@ -24,7 +24,9 @@
             "path": "../common"
         },
     ],
-
+    "tsc-alias": {
+        "resolveFullPaths": true
+    },
```

Comment:
Not sure why this is necessary.

Reply:
This allows me to import like so: `import { randomString } from 'common/util/random'` and `import { config } from 'discord-bot/constants/config'` without adding the `.js` extension normally required in es modules